lisa/trace.py

```diff
@@ -34,15 +32,43 @@
 import shlex
 from functools import reduce, wraps
 from collections.abc import Sequence
+from collections import namedtuple
+
+import numpy as np
+import pandas as pd
+
+import trappy
+import devlib
+from devlib.target import KernelVersion
 
 from lisa.utils import Loggable, HideExekallID, memoized, deduplicate, deprecate
 from lisa.platforms.platinfo import PlatformInfo
 from lisa.conf import SimpleMultiSrcConf, KeyDesc, TopLevelKeyDesc, StrList, Configurable
-import devlib
-from devlib.target import KernelVersion
 
 NON_IDLE_STATE = -1
 
+class TaskID(namedtuple('TaskID', ('pid', 'comm'), module=__name__)):
```

Comment:
`module` was introduced in Python 3.6 and thus fails with Travis.

Reply:
Now that I think about it, `module` is useless here since we have a proper class declaration anyway